.github/workflows/analysis_workflow.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       - name: Install ArcticDB[Testing]
         shell: bash -el {0}
         run: |
-          pip install arcticdb[Testing]
+          pip install arcticdb[Testing] "protobuf<5"
```

Comment:
Is this installing a wheel build from source? If so can't we just update the Testing section in `setup.cfg` with this?

If it's installing a wheel from PyPi then I agree we need this.

Reply:
it is from PyPi at the moment